release.nix

```diff
@@ -0,0 +1,13 @@
+with import ./. {};
```

Comment:
This will have to match up DEVOPS-690 to be built for multiple envs (staging, mainnet)